actions/websocket_actions.jsx

```diff
@@ -1425,6 +1427,8 @@ function handleThreadReadChanged(msg) {
                     },
                 );
             }
+        } else if (msg.broadcast.channel_id) {
+            handleAllThreadsInChannelMarkedRead(doDispatch, doGetState, msg.broadcast.channel_id, msg.data.timestamp);
```

Comment:
@ashishbhate, in what scenario does this occur? It would only affect threads that happen to be loaded for that channel (and very likely that threads would be missing).